content/javascript/concepts/objects/terms/isExtensible/isExtensible.md

```diff
@@ -0,0 +1,75 @@
+---
+Title: '.isExtensible()'
+Description: 'Returns a boolean value depending on whether a given object can have properties added to it.'
+Subjects:
+  - 'Web Development'
+  - 'Code Foundations'
+
+Tags:
+  - 'Methods'
+  - 'Objects'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The **`.isExtensible()`** is a static method that returns `true` if the given object can have properties added to it and `false` if the object cannot have properties added to it.
+
+## Syntax
+
+```pseudo
+Object.isExtensible(newObject);
+```
+
```

Comment:
Add a line describing the `newObject` parameter after the syntax block (Remember to give a line space between them).